### ISSUE TITLE
No GPS in UAV terminal anymore?

### ISSUE THREAD

**Issue by rlex** [KoffeinFlummi/AGM#2124]
Subj

**Comment by PabstMirror** [comment]
In AssignedItemsFix.sqf:

`showGPS     (AGM_isGPSEnabled     && {"ItemGPS"     in _items || {cameraOn == getConnectedUAV AGM_player}});`

I guess in vanilla a3 all of the 3 different UAV terminals (`X_UavTerminal`) offer GPS functionality that we take away.

**Comment by esteldunedain** [comment]
If they inherit from ItemGPS we should test for that instead of requiring and specific "ItemGPS" item. Also true for mods that replace the gps by tablets

**Comment by rlex** [comment]
Indeed, with AGM removed UAV Terminals can act as GPS receiver, while with AGM GPS is completely disabled in terminal.

**Comment by kripto202** [comment]
tbh i like the idea of no gps on the uav terminals.

**Comment by BryanBoru** [comment]
My unit just went in and overrode AGM on that. It also took gps away from
cTab Androids and Tablets.

**Comment by commy2** [comment]
Not a bug, but a design decision.
